### PR TITLE
CNDE-1911

### PR DESCRIPTION
**Pull request by joshua-mills-cdw**
## Notes

The fact table for Interview F_Interview needs to be populated using the post process SQL Proc using the corresponding D_Inverview and D_Interview_Note dimensions.

## JIRA

- **Related story**: [CNDE-1911](https://cdc-nbs.atlassian.net/browse/CNDE-1911)

## Checklist

- [x] PR focuses on a single story.
- [ ] New unit tests added and ensured they pass.
- [ ] Service has been tested in local and it works as expected.
- [x] Documentation has been updated for this code change (if needed).
- [ ] Code follows the Java Coding Conventions (https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html).

## Types of changes

What types of changes does this PR introduces?

- [ ] Bugfix
- [x] New feature
- [ ] Breaking change

## Testing

- [ ] Does this PR has >90% code coverage?
- [ ] Is the screenshot attached for code coverage?
- [ ] Does the `gradle build` pass in your local? 
- [ ] Is the `gradle build` logs attached?

### CODE REVIEW

File: liquibase-service/src/main/resources/db/rdb_modern/routines/024-sp_f_interview_case_postprocessing-001.sql [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+
+        BEGIN TRANSACTION
+
+        SET
+            @PROC_STEP_NO = @PROC_STEP_NO + 1;
+        SET
+            @PROC_STEP_NAME = 'INSERT INTO dbo.F_INTERVIEW_CASE';
+
+        
+        INSERT INTO dbo.F_INTERVIEW_CASE (
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb_modern/routines/024-sp_f_interview_case_postprocessing-001.sql:181]: 👍

File: liquibase-service/src/main/resources/db/rdb_modern/routines/024-sp_f_interview_case_postprocessing-001.sql [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                IX_INTERVIEWEE_KEY,
+                INTERVENTION_SITE_KEY
+                FROM #F_INTERVIEW_CASE_INIT
+                WHERE D_INTERVIEW_KEY NOT IN (SELECT D_INTERVIEW_KEY FROM #F_INTERVIEW_CASE_N)) ficu
+            ON fic.D_INTERVIEW_KEY = ficu.D_INTERVIEW_KEY;
+
+        SELECT @RowCount_no = @@ROWCOUNT;
+
+
+        INSERT INTO [dbo].[job_flow_log]
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb_modern/routines/024-sp_f_interview_case_postprocessing-001.sql:255]: Completion step log 🗒️

Comment (joshua-mills-cdw) [reply liquibase-service/src/main/resources/db/rdb_modern/routines/024-sp_f_interview_case_postprocessing-001.sql:255]: Added, thank you 👍

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

**Pull request merged**